distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -545,6 +547,78 @@ type Decimal
     floor : Integer
     floor self = @Builtin_Method "Decimal.floor"
 
+    ## ALIAS int
+
+       Truncate a floating-point number to an integer by dropping the fractional
+       part. This is equivalent to "round-toward-zero".
+    truncate : Integer
+    truncate self = self.truncate_builtin
+
+    ## Round to a specified number of decimal places.
+
+       By default, rounding uses "asymmetric round-half-up", also known as
+       "round towards positive infinity." If use_bankers=True, then it uses
+       "round-half-even", also known as "banker's rounding".
+
+       Arguments:
+       - decimal_places: The number of decimal places to round to. Can be
+         negative, which results in rounding to positive integer powers of 10.
+         Must be between -15 and 15 (inclusive).
+       - use_bankers: Rounds mid-point to nearest even number.
+
+       ! Error Conditions
+
+         If `decimal_places` is outside the range -15..15 (inclusive), an
+         `Illegal_Argument` error is thrown.
+
+       ? Negative decimal place counts
+         Rounding to `n` digits can be thought of as "rounding to the nearest
+         multiple of 10^(-n)". For negative decimal counts, this results in
+         rounding to the nearest positive integer power of 10.
+
+       > Example
+         Round to the nearest integer.
+
+            3.3 . round == 3
+
+       > Example
+         Round to two decimal places.
+
+            3.1415 . round 2 == 3.14
+
+       > Example
+         Round to the nearest hundred.
+
+            1234.0 . round -2 == 1200
+
+       > Example
+         Use Banker's Rounding.
+
+            2.5 . round use_bankers=True == 2
+    round : Integer -> Integer | Decimal ! Illegal_Argument
```

Comment:
I think the docstring should clarify when the return type is `Decimal` and when it can be expected to be `Integer`.

Reply:
And we should have some tests confirming that what the docstring will be saying is actually accurate.

(keep in mind that `3.0 . should_equal 3` will succeed because in Enso Integers and Decimals are coerced when being compared (`3 == 3.0`), so we need some additional checks like 
```
10.5.round . should_be_a Integer
10.0 . round 2 . should_be_a Decimal
10.round 2 . should_be_a ??? # This is especially interesting - what if we round an 'already integer' to `>0` decimal places? I assume if the input is an integer, the output should _always_ be an integer. But we should clarify that and verify if it is indeed true.
```

Reply:
Added docstring. Added tests for return types. Integer.round always returns an integer, even for dp>0.